docs/application/vscode-ext/web.md

```diff
@@ -333,46 +333,103 @@ To debug application, follow these steps:
 
     ![Debugging you application in the Web Simulator](media/tizen_web18.png)
 
-### Run Design Editor
+### Quick Action Execution
+Right-click on the **project folder** to use the extension function in the menu.
 
-To run Design Editor, follow one of the below ways:
+For more information on each function, see:
+- [Tizen Web: Build Tizen App](#Build-Project)
+- [Tizen Web: Debug Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Install Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Make Exclude Build List](#Exclude-Build-List)
+- [Tizen Web: Run Tizen App](#Deploy-and-Run-Application-in-Emulator)
+
+![Design Editor launch](media/tizen_web34.png)
+
+## Tizen Web with Design Editor
 
-> **Note** 
-> 
-> You must select the **HTML extension file** in the file explorer.
+Visual Studio Code Extension for Tizen Web supports WYSIWYG Design Editor features such as **Preview**, **JavaScript Assistant**, **Structure View**, **New Page**, **HTML Assistant** , **Undo** , **Redo**.
```

Comment:
I have asked to add comma but not to remove "and".  
...**HTML Assistant** , **Undo** , and **Redo**.